database/interface.py

```diff
@@ -626,5 +534,6 @@ def get_harvest_records_by_job(self, job_id, facets=[], **kwargs):
         return self.pget_harvest_records(filter=text(filter_string), **kwargs)
 
     def get_harvest_records_by_source(self, source_id, facets=[], **kwargs):
-        filter_string = " AND ".join([f"harvest_source_id = '{source_id}'"] + facets)
+        filter_string = " AND ".join([f"harvest_source_id = '{source_id}'"] 
+                        + ["action != 'delete'"]+ facets)
```

Comment:
are the records now being retained as deletes, or is this to cleanup an earlier bug?

Reply:
no, it is just when a clear/delete operation in progress, any records marked with the 'delete' action should be excluded from the count when refreshing the page.